ksql-engine/src/main/java/io/confluent/ksql/util/KafkaTopicClientImpl.java

```diff
@@ -19,24 +19,28 @@
 import io.confluent.ksql.exception.KafkaResponseGetFailedException;
 import io.confluent.ksql.exception.KafkaTopicException;
 import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.DeleteTopicsResult;
 import org.apache.kafka.clients.admin.NewTopic;
 import org.apache.kafka.clients.admin.TopicDescription;
+import org.apache.kafka.common.KafkaFuture;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.util.Arrays;
 import java.util.Collection;
 import java.util.Collections;
+import java.util.List;
 import java.util.Map;
 import java.util.Set;
 import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
 
 public class KafkaTopicClientImpl implements KafkaTopicClient {
   private static final Logger log = LoggerFactory.getLogger(KafkaTopicClient.class);
-  private final Map<String, Object> adminClientConfig;
+  private final AdminClient adminClient;
```

Comment:
Is 4.0.1 using kafka 11.0.1 where the adminclient is fixed?

Reply:
we use 4.0.0 and it has Kafka 1.0

Reply:
Yes, the adminclient should be fixed there. I also reimported your changes to use singleton adminclinent